plugins/candela/web_client/views/CandelaWidget.js

```diff
@@ -44,61 +45,65 @@ var CandelaWidget = View.extend({
     },
 
     render: function () {
-        if (this.accessLevel >= AccessType.READ && this.item.get('name').toLowerCase().endsWith('.csv')) {
-            this.$el.html(CandelaWidgetTemplate({
-                components: this._components
-            }));
-            this.parametersView.setElement($('.g-item-candela-parameters'));
-            datalib.csv(this.item.downloadUrl(), (error, data) => {
-                if (error) {
-                    let info = {
-                        text: 'An error occurred while attempting to read and ' +
-                              'parse the data file. Details have been logged in the console.',
-                        type: 'danger',
-                        timeout: 5000,
-                        icon: 'attention'
-                    };
-                    events.trigger('g:alert', info);
-                    console.error(error);
-                    return;
-                }
+        let cleanup = function () {
```

Comment:
Though it doesn't matter in terms of behavior here, it's probably cleaner if lambdas are defined as arrow functions unless otherwise necessary.